packages/esm-patient-labs-app/src/test-results/overview/common-datatable.component.tsx

```diff
@@ -66,17 +66,14 @@ const CommonDataTable: React.FC<CommonDataTableProps> = ({ title, data, descript
             </TableHead>
             <TableBody>
               {rows.map((row, i) => (
-                <TypedTableRow key={row.id} interpretation={data[i]?.interpretation} {...getRowProps({ row })}>
+                <TypedTableRow key={row.id} interpretation={data[i]?.value?.interpretation} {...getRowProps({ row })}>
                   {row.cells.map((cell) => {
-                    return (
-                      <TableCell
-                        className={
-                          cell.value?.interpretation ? styles[interpretationToCSS[cell.value.interpretation]] : ''
-                        }
-                        key={cell.id}
-                      >
-                        {cell.value?.value ?? cell.value}
+                    return cell.value?.interpretation ? (
+                      <TableCell className={styles[interpretationToCSS[cell.value.interpretation]]} key={cell.id}>
+                        <p>{cell.value?.value ?? cell.value}</p>
```

Comment:
Maybe this should be a span instead to avoid a validateDomNesting warning because a paragraph is a block-level element?